compiler/runtime/OMRCodeMetaDataManager.cpp

```diff
@@ -294,6 +295,12 @@ CodeMetaDataManager::insertMetaDataRangeInHash(
       return 1;
       }
 
+   // Don't insert zero sized ranges.
+   if (startPC == endPC)
+      {
+      return 1;
+      }
+
```

Comment:
What's wrong with inserting zero-sized ranges?  Will this code not function?  Suppose someone wants to insert metadata for a particular code address in their runtime (not sure why, but they might have a need for it).  If we require a range of at least one to cover a code range then it should be documented somewhere in the API.